src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -248,19 +261,21 @@ const getAvailablePaymentMethods = (
   // when available, unshift ACH since it's the first option we want to offer
   if (availablePaymentMethods.includes("US_BANK_ACCOUNT")) {
     paymentMethods.unshift(
-      <BorderedRadio
+      <RadioWithLabel
+        id="payment-method"
         key="US_BANK_ACCOUNT"
         value={(paymentMethod = "US_BANK_ACCOUNT")}
         label={
           <>
             <InstitutionIcon fill="green100" />
             <Text ml={0.5}>Bank transfer</Text>
-            <USBankOnlyLabel ml={0.5} variant="xs">
-              US bank account only
-            </USBankOnlyLabel>
           </>
         }
-      />
+      >
+        <USBankOnlyLabel ml={0.5} variant="xs">
+          US bank account only
+        </USBankOnlyLabel>
```

Comment:
**question:** Does moving this component out of the `label` attribute fix the issue, or does the new set of CSS rules in the styled component fix the issue? (Or is it both?)

Reply:
That make sense. Thank you for the explanation!